js/src/collections/map/indicators_by_target.js

```diff
@@ -0,0 +1,22 @@
+var Handlebars = require('handlebars');
+var _ = require('lodash');
+var CartoDBCollection = require('../../lib/cartodb_collection.js');
+var CONFIG = require('../../../config.json');
+
+var SQL = Handlebars.compile(require('../../queries/map/indicators_by_target.hbs'));
+
+var IndicatorConfigs = CartoDBCollection.extend({
+  user_name: CONFIG.cartodb.user_name,
```

Comment:
This is not necessary set this variable anymore. It's setted in the CDB Collection class, and as you extend of it, you already got it. Other way, you need to setup the table name here.